CONTRIBUTING.md

```diff
@@ -25,9 +25,9 @@ We’re especially interested in learning what you need to implement the [Standa
 
 ## Public organizations
 
-We’re currently looking for innovative public organizations to join us as founding members. As a founding member, you’ll have the chance to shape the Foundation’s processes and governance. Your code will be among the first accepted for Foundation stewardship. Lead the community of enterprising public organizations creating public code to power their digital transformations.
+We’re currently looking for innovative public organizations to [join us as founding members](https://www.publiccode.net/founding-membership). As a founding member, you’ll have the chance to shape the Foundation’s processes and governance. Your code will be among the first accepted for Foundation stewardship.
```

Comment:
Make sure this link exists before pulling this pull request. Travis now throws an error because it is broken.

Reply:
http://publiccode.net/membership/founding-membership is now live